pwa/src/service-worker/service-worker.js

```diff
@@ -143,7 +144,8 @@ export function setUpServiceWorker() {
         ({ url }) =>
             PRODUCTION_ENV &&
             urlMeetsAppShellCachingCriteria(url) &&
-            fileExtensionRegexp.test(url.pathname),
+            fileExtensionRegexp.test(url.pathname) &&
+            !/\.(json|action)/.test(url.pathname), // don't SWR these
```

Comment:
LIBS-357

Reply:
There are other extensions we should probably except here - wouldn't it be easier to use `/api` and other base paths instead of trying to identify extensions?

Reply:
That could work; do you have a sense of what those rules might be?

I'm wondering if there might be requests to external URLs for `json`, for example

Reply:
Addressed this with a different approach here: https://github.com/dhis2/app-platform/pull/762